ajax/libs/kurento-client/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "kurento-client",
+  "description": "Media API for the Kurento Web SDK",
+  "license": "Apache-2.0",
+  "filename": "kurento-client.min.js",
+  "keywords": [
+    "API",
+    "Kurento",
+    "WebRTC"
+  ],
+  "author": "Kurento <info@kurento.com> (https://www.kurento.org)",
+  "homepage": "https://www.kurento.org",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Kurento/kurento-client-js.git"
+  },
+  "npmName": "kurento-client",
+  "npmFileMap": [
+    {
+      "basePath": "src/main/resources/META-INF/resources/js",
+      "files": [
+        "kurento-client+(.|_)!(cov*)"
+      ]
+    }
+  ],
+  "version": "6.6.0"
```

Comment:
Don't know if this is absolutely necessary but should we keep the version field near the top since the convention is to always put auto-update in the end of `package.json`?